e2e/README.md

```diff
@@ -0,0 +1,29 @@
+## How to execute E2E?
+
+1. Create a VPC Cluster
+2. Export the KUBECONFIG
+   In kube config file use abosulte path for `certificate-authority`, `client-certificate` and `client-key`
+3. Deploy the Driver (with SC)
+4. Export enviornment variables
```

Comment:
Minor Enhancement: we call add all the env variables mentioned in `Step 4 ` to a simple file and source it before running e2e. This will  save time for whoever is trying to run the e2es.